html/rankings/growth-predictions-list.html

```diff
@@ -0,0 +1,548 @@
+{% extends "template.html" %}
+{% block page_title %}Growth Projections - Latin America{% endblock %}
+{% block page_description %}New growth projections predict the rise of India, East Africa and fall of oil economies{% endblock %}
+
+{% block css %}
+  <link rel="stylesheet" href="{{ STATIC_URL }}css/rankings.css">
+{% endblock %}
+
+{% block content %}
+
+<div class="sidebar">
+  <h3 class="sidebar__head"><a href="rankings/country/">Country Rankings</a></h3>
+  <h4 class="sidebar__subhead">Rank countries based on their economic complexity for a given year</h4>
+  <h3 class="sidebar__head"><a href="rankings/product/">Product Rankings</a></h3>
+  <h4 class="sidebar__subhead">Rank products in the HS4 classification based on their complexity for a given year</h4>
+  <h3 class="sidebar__head"><a href="rankings/growth-predictions-list/">Growth Predictions</a></h3>
+  <h4 class="sidebar__subhead">Rank countries based on their predicted annual growth through 2023</h4>
+  <h3 class="sidebar__head"><a href="rankings/growth-predictions/">Summary: International Growth Predictions</a></h3>
+  <h4 class="sidebar__subhead">New Growth Projections Predict the Rise of India, East Africa and Fall of Oil Economies</h4>
+  <h3 class="sidebar__head"><a href="rankings/growth-predictions-latin-america/">Summary: Latin America Growth Predictions</a></h3>
+  <h4 class="sidebar__subhead">New Growth Projections Predict the Continued, Uneven Slowdown of Latin America</h4>
+  <h3 class="sidebar__head">Download</h3>
+  <ul class="sidebar__list">
+    <li class="sidebar__item"><a href="/media/growth_projections/country_rankings_and_projections_2013.csv">&raquo; Download 2013 data and growth projections</a></li>
+    <li class="sidebar__item"><a href="rankings/country/download/">&raquo; Download all years data</a></li>
+  </ul>
+</div>
+
+<article class="news__article">
+
+  <header class="news__header">
+    <h3 class="news__head">Growth Projections, 2013</h3>
```

Comment:
:speak_no_evil: not a bug, but this title makes it sound like the story is 2 years old. Would it be better if we said something like: "Growth Projects based on 2013 Annual Exports?" Or is that only better in my head?